datanode/networkhistory/store/store.go

```diff
@@ -226,12 +228,16 @@ func (p *Store) Stop() {
 	p.log.Info("Cleaning up network history store")
 	if p.ipfsNode != nil {
 		p.log.Info("Closing IPFS node")
-		_ = p.ipfsNode.Close()
+		if err := p.ipfsNode.Close(); err != nil {
+			p.log.Errorf("Failed to close IPFS node:%s", err)
```

Comment:
We usually used structured logs instead of formtated ones like so:
```suggestion
			p.log.Error("Failed to close IPFS node", logging.Error(err))
```